Examples/Echo/RUNME

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+#
+# Use this to run the swift-proto generator
+#
+protoc echo.proto --swift_out=. --swift_opt=Visibility=Public --plugin=/Users/tim/bin/protoc-gen-swift
```

Comment:
It's [not that new](https://github.com/apple/swift-protobuf/blame/d5ccb9558c0083f7fbe686db0889030560d2c5fe/Documentation/PLUGIN.md), but it wasn't needed for 0.9.24, so grpc-swift wasn't using it.